modules/microshift-manifests-upgrade-scenario-ostree.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-upgrade-scenario-ostree_{context}"]
+== Manifest upgrade scenario for OSTree systems
+Use the following procedure in the upgrade scenario for OSTree systems where some resources need to be removed while retaining others to preserve data.
```

Comment:
Blank line needed after title. Title needs to be at `=` level.

Reply:
done